keps/sig-network/4427-relaxed-dns-search-validation/kep.yaml

```diff
@@ -0,0 +1,37 @@
+title: Relaxed DNS search string validation
+kep-number: 4427
+authors:
+  - "@sethev"
+owning-sig: sig-network
+status: provisional
+creation-date: 2024-01-22
+reviewers:
+  - TBD
```

Comment:
@aojea or @danwinship - got time to serve as reviewer?

@SergeyKanzhelev or @dchen1107 - can someone from node take a look?

Reply:
Please update this to @danwinship